SU-HSDO/stanford_mrc
# MRCD8-230 Event Series Term changes

# READY FOR REVIEW

# Summary
- Changed the exposed filters
- Changed the titles on the event series blocks dynamically
- Changed order of blocks on term pages
- Added class to read more button
- Wrapped view rows in a div to fix `decanter-width-*` classes on view rows

# Needed By (Date)
- end of sprint

# Urgency
- High

# Steps to Test

1. a patch was applied on core so ensure you have pull and do latest composer update
2. sync from production
3. update database
4. optionally enable devel_generate and create lots of events, news and visitors
5. view the event series pages and ensure the changes in the summary above are visible.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
